sdk/keyvault/azure-keyvault-administration/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 4.2.0 (2022-09-13)
```

Comment:
Ah, here and elsewhere, please update to 9/19 or 9/20 (we plan to slip to 9/20 for the announcement).